cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/batch/BatchSQLEngineAdapter.java

```diff
@@ -189,7 +224,7 @@ public SQLDataset pushInternal(String datasetName,
    */
   @SuppressWarnings("unchecked,raw")
   public <T> SQLEngineJob<JavaRDD<T>> pull(SQLEngineJob<SQLDataset> job,
-                                       JavaSparkContext jsc) {
+                                           JavaSparkContext jsc) {
```

Comment:
As you now have it as a field, you don'e need it as parameter, do you? Same below